app/templates/direct-award/did-you-award-contract.html

```diff
@@ -0,0 +1,70 @@
+{% extends "toolkit/layouts/_base_page.html" %}
+
+{% block page_title %}Did you award a contract? - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+        "link": url_for('main.index'),
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": url_for('external.buyer_dashboard'),
+        "label": "Your account"
+      },
+      {
+        "link": url_for('direct_award.saved_search_overview', framework_framework=framework.framework),
+        "label": "Your saved searches"
+      },
+      {
+        "link": url_for('direct_award.view_project', framework_framework=framework.framework, project_id=project.id),
+        "label": project.name
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="did-you-award-contract-page">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <header class="page-heading">
+        <h1>
+          Did you award a contract for '{{project.name}}'?
```

Comment:
One of our old designers has got us all in the habit of only using u+2019 single right quotes (`&rsquo;` or `’`) everywhere in content. We're fairly consistent about this and it was a big thing a while ago, so we try to keep this alive for the time being.

Reply:
We actually just use the right quotes, but I'm sure it's fine this time. 😛  🤷‍♂️

Reply:
Yeah Laurence scoured the code and corrected me.

I have _no idea_ why I had that obviously illogical decision in my head.

Ignore it!